src/Sulu/Bundle/WebsiteBundle/Resolver/StructureResolverInterface.php

```diff
@@ -26,5 +26,5 @@ interface StructureResolverInterface
      *
      * @return array
      */
-    public function resolve(StructureInterface $structure, bool $loadExcerpt = true);
+    public function resolve(StructureInterface $structure, bool $loadExcerpt = true, array $includedProperties = null);
```

Comment:
TODO: add parameter in a non-breaking way via c 🙂

Reply:
If we go with the mapping maybe which would include also the `excerpt` data we could go with:

```
/**
 * @param bool|array $includedProperties
 */
public function resolve(StructureInterface $structure, $includedProperties = true)
{
      if (is_bool($includedProperties)) {
      
      }
}
```

Would be a bc break.

Reply:
ohh didn't see this is the StructureResolver and not the extension 🤔